Header header

#The name of the frame of the gridboard
string name

#The pose of the gridboard in frame from header
geometry_msgs/Pose pose

#The Aruco gridboard
ArucoGridboard marker
